src/pretix/static/pretixcontrol/scss/main.scss

```diff
@@ -22,6 +22,11 @@
 .bootstrap-datetimepicker-widget table td span {
     line-height: 1.8em;
 }
+/* fix leaflet overlay datetime-picker, see https://github.com/pretix/pretix/issues/2480 */
+.leaflet-top,
+.leaflet-bottom {
+  z-index: 900;
+}
```

Comment:
Maybe a better location would be here?
https://github.com/pretix/pretix/blob/master/src/pretix/static/pretixcontrol/scss/_forms.scss#L284

Reply:
Thanks, I moved it there.